src/Materials/Node/nodeMaterialBlock.ts

```diff
@@ -706,6 +706,7 @@ export class NodeMaterialBlock {
         serializationObject.id = this.uniqueId;
         serializationObject.name = this.name;
         serializationObject.comments = this.comments;
+        serializationObject.visibleInInspector = this.visibleInInspector;

```

Comment:
You should also update the `InputBlock._dumpPropertiesCode` method to remove the handling of `visibleInInspector` and move it to `NodeMaterialBlock._dumpPropertiesCode` instead.